### PR TITLE
[NewUI-flat] New deposit ether modal UI.

### PR DESCRIPTION
**Pull request by danjm**
This PR implements the new design for the deposit ether modal. It is missing a working shapeshift link (waiting for reply on slack).

screen shot 2017-12-02 at 3 04 05 am
--
screen shot 2017-12-02 at 3 04 24 am
--
screen shot 2017-12-02 at 3 04 50 am
--
buymodal

### CODE REVIEW

**Comment by chikeichan** [comment]
Let's make mobile modal fullscreen

**Comment by danjm** [comment]
022b209d New deposit modal full screen on mobile, and other style fixes.

revisedmobiledepositmodal

Full screen stays consistent:

screen shot 2017-12-07 at 1 22 33 pm

**Comment by danjm** [comment]
I have made changes to include the shapeshift modal. The UI changes are captured below:

buywithshapeshift-actualbuy
-
buywithshapeshift-mobile
-
buywithshapeshift-browser

@cjeria Could you review the above screen shots and let me know if you'd like any changes made. I have ported the design from the Mascara first time flow, but had to make some design/UX decisions when integrating with the buy modal and I'm happy to make changes.

**Comment by danjm** [comment]
@alextsg d4565eac

screen shot 2018-01-10 at 3 15 57 pm

**Comment by cjeria** [comment]
@danjm this looks good from what I can see from the gifs. What happens after I go through the entire flow of trading a crypto for eth? Are there other screens we should look at in the flow like a success screen?

**Comment by danjm** [comment]
> @danjm this looks good from what I can see from the gifs. What happens after I go through the entire flow of trading a crypto for eth? Are there other screens we should look at in the flow like a success screen?

@cjeria No, there is no success screen for a shapeshift trade. See the first of the three gifs I posted. A shapeshift transaction will appear in the transaction list. Once the user actually sends bitcoin to the address, then the status of that transaction will update.

File: ui/app/components/shapeshift-form.js [hunk 1]
```diff
@@ -1,308 +1,243 @@
...
+  return h('div.shapeshift-form', {}, [
 
-  return h('datalist#coinList', {
-    onClick: (event) => {
-      event.preventDefault()
-    },
-  }, list)
-}
+    h('div.shapeshift-form__deposit-instruction', [
+      'Deposit your BTC to the address bellow:',
```
Comment (alextsg) [inline ui/app/components/shapeshift-form.js]: typo: "bellow"

File: ui/app/components/modals/deposit-ether-modal.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+      h('div.deposit-ether-modal__header__close', {
+        onClick: () => {
+          this.setState({ buyingWithShapeshift: false })
+          this.props.hideModal()
+        },
+      }),
+
+    ]),
+
+    // h('div.deposit-ether-modal__buy-rows-container', [
```
Comment (alextsg) [inline ui/app/components/modals/deposit-ether-modal.js]: Delete unless it's a placeholder for possible future changes.

File: ui/app/css/itcss/components/modal.scss [hunk 3]
```diff
@@ -598,4 +598,263 @@
...
+      font-size: 2em;
+      color: $white;
+      position: absolute;
+      top: 20.8px;
+      right: 28px;
+      cursor: pointer;
+    }
+  }
+
+  // &__buy-rows-container {
```
Comment (alextsg) [inline ui/app/css/itcss/components/modal.scss]: Delete unneeded styles

**Review by alextsg — CHANGES_REQUESTED** [review]

**Comment by danjm** [comment]
@alextsg fixed issues you identified here: 29389936

Also removed some code intended to allow us to identify which accounts created a shape shift transaction. Something like this is needed to ensure that a shapeshift tx created in one account doesn't show up in another. But my impementation here creates other problems. We'll have to discuss a better solution and implement in a future PR.

**Review by alextsg — APPROVED** [review]

**Pull request merged**